form/2d/p5js/closethetriangle/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: [Jakob von Kietzell]
+author-url: [https://incom.org/profil/7939]
+title: [closethetriangle]
+tags: [P5.js,[FORM], [2D], Law of Closure]
```

Comment:
no inner `[]` here the only `[]` are the outer ones. Tags like "form" or "2d" are written in lowercase. Laws are written like titles.